docs/user/filesystem.md

```diff
@@ -195,29 +195,283 @@ The contents of multiple pools/containers will be accessible via this
+$  cat /tmp/scratch_fs-root_dfuse.mount
+# Automatically generated by systemd-fstab-generator
+
+[Unit]
+SourcePath=/etc/fstab
+Documentation=man:fstab(5) man:systemd-fstab-generator(8)
+Before=local-fs.target
+After=daos_agent.service
+Requires=daos_agent.service
+
+[Mount]
+Where=/scratch_fs/root_dfuse
+What=dfuse
+Type=fuse3
+Options=pool=admin_pool,container=admin_cont,auto,x-systemd.requires=daos_agent.service
+$  cp -rf /tmp/scratch_fs-root_dfuse.mount  /usr/lib/systemd/system/
+$  systemctl daemon-reload
+$  systemctl status scratch_fs-root_dfuse.mount
+● scratch_fs-root_dfuse.mount - /scratch_fs/root_dfuse
+   Loaded: loaded (/etc/fstab; generated)
+   Active: inactive (dead) since Fri 2022-09-23 15:55:33 UTC; 1min 50s ago
+    Where: /scratch_fs/root_dfuse
+     What: dfuse
+     Docs: man:fstab(5)
+           man:systemd-fstab-generator(8)
+
+Sep 23 15:55:33 wolf-170.wolf.hpdd.intel.com systemd[1]: scratch_fs-root_dfuse.mount: Succeeded.
+$  systemctl start scratch_fs-root_dfuse.mount
+$  df -h | grep fuse
+dfuse                         537G  5.1G  532G   1% /scratch_fs/root_dfuse
+$  ls -l /scratch_fs/root_dfuse/
+total 0
+$  systemctl status scratch_fs-root_dfuse.mount
+● scratch_fs-root_dfuse.mount - /scratch_fs/root_dfuse
+   Loaded: loaded (/etc/fstab; generated)
+   Active: active (mounted) since Fri 2022-09-23 15:57:53 UTC; 31s ago
+    Where: /scratch_fs/root_dfuse
+     What: dfuse
+     Docs: man:fstab(5)
+           man:systemd-fstab-generator(8)
+    Tasks: 63 (limit: 1648282)
+   Memory: 51.5M
+   CGroup: /system.slice/scratch_fs-root_dfuse.mount
+           └─4173 dfuse /scratch_fs/root_dfuse -o rw pool=admin_pool container=admin_cont dev suid
+
+Sep 23 15:57:52 wolf-170.wolf.hpdd.intel.com systemd[1]: Mounting /scratch_fs/root_dfuse...
+Sep 23 15:57:53 wolf-170.wolf.hpdd.intel.com systemd[1]: Mounted /scratch_fs/root_dfuse.
+$  systemctl stop scratch_fs-root_dfuse.mount
+$  systemctl status scratch_fs-root_dfuse.mount
+● scratch_fs-root_dfuse.mount - /scratch_fs/root_dfuse
+   Loaded: loaded (/etc/fstab; generated)
+   Active: inactive (dead) since Fri 2022-09-23 15:58:32 UTC; 2s ago
+    Where: /scratch_fs/root_dfuse
+     What: dfuse
+     Docs: man:fstab(5)
+           man:systemd-fstab-generator(8)
+    Tasks: 0 (limit: 1648282)
+   Memory: 540.0K
+   CGroup: /system.slice/scratch_fs-root_dfuse.mount
+
+Sep 23 15:57:52 wolf-170.wolf.hpdd.intel.com systemd[1]: Mounting /scratch_fs/root_dfuse...
+Sep 23 15:57:53 wolf-170.wolf.hpdd.intel.com systemd[1]: Mounted /scratch_fs/root_dfuse.
+Sep 23 15:58:32 wolf-170.wolf.hpdd.intel.com systemd[1]: Unmounting /scratch_fs/root_dfuse...
+Sep 23 15:58:32 wolf-170.wolf.hpdd.intel.com systemd[1]: scratch_fs-root_dfuse.mount: Succeeded.
+Sep 23 15:58:32 wolf-170.wolf.hpdd.intel.com systemd[1]: Unmounted /scratch_fs/root_dfuse.
+$
+```
+
+#### 4> systemd to auto mount the dfuse container after system power cycle or reboot.
```

Comment:
I am not sure to understand the difference with the previous section ... could you please elaborate?